lxc/storage_volume.go

```diff
@@ -413,6 +415,7 @@ func (c *cmdStorageVolumeCopy) Run(cmd *cobra.Command, args []string) error {
 		args.Name = dstVolName
 		args.Mode = mode
 		args.VolumeOnly = false
+		args.TargetProject = c.flagTargetProject
```

Comment:
Per my earlier comment, we shouldn't be using the Args struct for this. Instead we should do it as we do for containers (see lxc/copy.go and how it uses flagTargetProject).

So that would be setting up the dest remote using `dest = dest.UseProject(c.flagTargetProject)` and then let the code in `client/` figure out what to set in the source struct based on what the source and destination is.